.buildkite/pipeline.yml

```diff
@@ -51,4 +51,6 @@ steps:
     command: buildkite-agent pipeline upload .buildkite/pipeline-e2e-tests.yml
 
   - label: ":buildkite:"
+    depends_on:
+      - "operator-image-build"
```

Comment:
with this change, do we need this ?
https://github.com/elastic/cloud-on-k8s/blob/dcea160ed9540b9c1da3c63f43cdbb84705610b9/.buildkite/pipeline-release.yml#L10

Reply:
There is no longer a dependency between the build and the helm release, but we still don't want to release the helm charts before the docker images referenced therein are released.